spec/rails-2.3/spec/spec_helper.rb

```diff
@@ -26,11 +18,19 @@
 Dir[File.expand_path(File.join(File.dirname(__FILE__),'support','**','*.rb'))].each {|f| require f}
 
 # Run the migrations
+ConsulMigration = ActiveRecord::Migration
 print "\033[30m" # dark gray text
 ActiveRecord::Migrator.migrate("#{Rails.root}/db/migrate")
 print "\033[0m"
 
+module ControllerSpecHelpers
+  def wrap_params(params)
```

Comment:
Can we add a comment for future devs explaining why we need this in controller specs.